test/main.test.js

```diff
@@ -57,33 +43,34 @@ describe("main", () => {
     it(
       gwt`cold start | game is loaded | first player rolls dice and ends turn`,
       () => {
-        require("../entities/Game")(eventBusEmitter, mockUI, gameState);
-
         // arrange
+        const promptStub = sinon.stub(PlayerActions, "prompt");
+        // highest rolling player
```

Comment:
Put this comment beside the below two lines, else add a comment before line 51 with `// player turn` or something ?

Reply:
done!